packages/wallet/backend/src/paymentPointer/controller.ts

```diff
@@ -73,6 +74,27 @@ export class PaymentPointerController implements IPaymentPointerController {
     }
   }
 
+  getExternalPaymentPointer = async (
+    req: Request,
+    res: CustomResponse<ExternalPaymentPointer>,
+    next: NextFunction
+  ) => {
+    try {
+      if (!req.params.url) {
```

Comment:
have a schema to validate that it is a string in the correct format instead of this check maybe

Reply:
<img width="424" alt="image" src="https://github.com/interledger/testnet/assets/111863110/fe34920d-df49-4d6d-95ef-403840df99d1">

Reply:
would this be fine? or would you prefer external/payment-pointers (i don't think there will be any other external stuff though that would fall under the first / category)